packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -413,7 +435,11 @@ contract ChugSplashManager is Owned {
     function getProxyByName(string memory _name) public view returns (address payable) {
         return (
             payable(
-                Create2.compute(address(this), keccak256(bytes(_name)), type(Proxy).creationCode)
+                Create2.compute(
+                    address(this),
+                    keccak256(bytes(_name)),
+                    abi.encodePacked(type(Proxy).creationCode, abi.encode(address(this)))
```

Comment:
Changed the `_bytecode` argument in `Create2` from `type(Proxy).creationCode` to `abi.encodePacked(type(Proxy).creationCode, abi.encode(address(this)))` since this argument is supposed to be the creation code concatenated with the ABI-encoded constructor arguments ([source](https://docs.soliditylang.org/en/latest/control-structures.html?highlight=create2#salted-contract-creations-create2))